java/src/main/kotlin/io/spine/protodata/java/ClassName.kt

```diff
@@ -101,6 +101,21 @@ public class ClassName(
     public fun nested(simpleClassName: String): ClassName =
         ClassName(packageName, simpleNames + simpleClassName)
 
+    /**
+     * Obtains a new `ClassName` with the last element removed from the list of simple names.
+     *
+     * The method is useful for obtaining names for outer classes,
+     * e.g. `Message` from `Message.Builder`.
+     *
+     * @throws IllegalStateException when the current class name is the top level class name.
+     */
+    public fun outer(): ClassName {
```

Comment:
If would be easier to deal with this API if we return `ClassName?` instead of throwing the exception.

For example, with a nullable output it would be possible to climb up the nested classes. With the current approach you'd have to catch exceptions to do that.